src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Consider using a descriptive constant for the discount value `50` instead of the hardcoded value. This will make your code more maintainable and easier to understand.